synapse/lib/stormtypes.py

```diff
@@ -7328,6 +7350,7 @@ def getObjLocals(self):
             'del': self._methViewDel,
             'get': self._methViewGet,
             'list': self._methViewList,
+            'bylayer': self._methViewByLayer,
```

Comment:
The name `bylayer` feels a bit like you should specify a layer and get a list of views that contain it.  Maybe we could think of a better name?

Reply:
OBE